mesh/mesh.cpp

```diff
@@ -7131,29 +7131,27 @@ Table *Mesh::GetEdgeVertexTable() const
 
 Table *Mesh::GetVertexToElementTable()
 {
-   int i, j, nv, *v;
-
    Table *vert_elem = new Table;
 
    vert_elem->MakeI(NumOfVertices);
 
-   for (i = 0; i < NumOfElements; i++)
+   for (int i = 0; i < NumOfElements; i++)
    {
-      nv = elements[i]->GetNVertices();
-      v  = elements[i]->GetVertices();
-      for (j = 0; j < nv; j++)
+      int nv = elements[i]->GetNVertices();
```

Comment:
Could be also `const int`.

Reply:
Done in [45e2636](https://github.com/mfem/mfem/pull/4327/commits/45e2636921d7bfdea22f8120d379fb1ffc3a8671).

Reply:
Perfect 👍